openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -121,6 +122,19 @@ def _create_from_course(cls, course):
             end = ccx.due
             max_student_enrollments_allowed = ccx.max_student_enrollments_allowed
 
+        # Try to create a thumbnail of the course image. If this fails for any
+        # reason (weird format, non-standard URL, etc.), just fall back to the
+        # old behavior of using the originally provided course_image_url().
+        try:
+            course_image_url_ = create_course_image_thumbnail(course, THUMBNAIL_DIMENSIONS)
```

Comment:
The Course About page only displays at 338 x 180. I'm not averse to making another field, but does it make sense just to make a couple? The Course About API might want a bigger pic, but it definitely doesn't want the 12MB monster images. If so, maybe I could just make a new table with a couple of columns for low and high res versions (with potentially more coming later)?

If so, what resolution makes sense for the About API? A strict doubling at 750 x 400?